mentorship ios/Views/Tasks/TaskComments.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  TaskComments.swift
+//  Created on 15/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import SwiftUI
+
+struct TaskComments: View {
+    let taskCommentsService: TaskCommentsService = TaskCommentsAPI()
+    @EnvironmentObject var taskCommentsVM: TaskCommentsViewModel
+    let taskID: Int
+    let taskName: String
+    
+    func fetchComments() {
+        taskCommentsService.fetchTaskComments(reqID: taskCommentsVM.reqID, taskID: taskID) { comments in
+            self.taskCommentsVM.isLoading = false
+            self.taskCommentsVM.taskCommentsResponse = comments
+        }
+    }
+    
+    func postComment() {
+        self.taskCommentsService.postTaskComment(
+            reqID: self.taskCommentsVM.reqID,
+            taskID: self.taskID,
+            commentData: self.taskCommentsVM.newComment
+        ) { resp in
+            // update view model with responses
+            self.taskCommentsVM.postCommentResponse = resp
+            // if post comment successful, update comments and clear text field
+            if resp.success {
+                self.fetchComments()
+                self.taskCommentsVM.newComment.comment = ""
```

Comment:
question: should we clear self.taskCommentsVM.postCommentResponse also here?

Reply:
Not sure about this point, clear in the sense?